package.json

```diff
@@ -58,7 +58,7 @@
     "shx": "0.3.4",
     "sinon": "11.1.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
+    "typescript": "^5.0.3",
```

Comment:
@mshanemc I was wondering if we need to do more verification around a move to v5? I took a quick peek at swc/core and the project is still using version 4.